conf/application-example.conf

```diff
@@ -35,4 +35,10 @@ securesocial{
   # for local development you probably don't have an https proxy set up
   # if you do: remove this line:
   ssl=false
-}
\ No newline at end of file
+}
+
+rabbitmq {
+  host = "rabbit000.dev.yetu.me"
+}
+
+riak.production.host = "riak000.dev.yetu.me"
```

Comment:
it is not necessary remember? the test line was just necessary because I was running from my IDE without -D parameter.. but if you run from sbt commands, that are the recommended way.. it isn't necessary..